Intersect.Server.Core/Entities/BankInterface.cs

```diff
@@ -418,9 +418,14 @@ public bool TryWithdrawItem(Item? slot, int bankSlotIndex, int quantityHint, int
                             break;
                         }
 
-                        var slotIndexToRemoveFrom = slotIndicesToRemoveFrom[nextSlotIndexToRemoveFrom++];
+                        var slotIndexToRemoveFrom = slotIndicesToRemoveFrom[nextSlotIndexToRemoveFrom];
                         var sourceSlot = sourceSlots[slotIndexToRemoveFrom];
+                        if (sourceSlot.Quantity <= maximumStack)
+                        {
+                            nextSlotIndexToRemoveFrom++;
```

Comment:
Will this work okay if the player tries to withdraw more items than they have open inventory slots?

Reply:
Yes, seems to work well.